worker/uniter/operation/state_test.go

```diff
@@ -74,12 +80,14 @@ var stateTests = []struct {
 	},
 	// RunAction operation.
 	{
+		description: "run action without action id",
 		st: operation.State{
 			Kind: operation.RunAction,
 			Step: operation.Pending,
 		},
 		err: `missing action id`,
 	}, {
+		description: "run action without action id",
```

Comment:
+1 on the description additions; very helpful.
Though, this one looks like an errant copy-paste from the one prior.

Reply:
:+1:
When I have to figure out which test is running, descriptions help me understand *what* is going on, and very much *which* one to fix. Much better than "test 5 failed"